salt/states/pkg.py

```diff
@@ -582,22 +601,28 @@ def _find_install_targets(name=None,
                     and not reinstall \
                     and not pkg_verify:
                 # The package is installed and is the correct version
-                return {'name': name,
-                        'changes': {},
-                        'result': True,
-                        'comment': 'Version {0} of package \'{1}\' is already '
-                                   'installed'.format(version, name)}
+                ret = {'name': name,
+                       'changes': {},
+                       'result': True,
+                       'comment': 'Version {0} of package \'{1}\' is already '
```

Comment:
BTW, sometimes we use single quotes, sometimes double quotes, sometimes none at all in these situations. Maybe at least we can get rid of those ugly back-slashes in the code. So either one looks OK to me:

```python
'Version {0} of package {1} is already '  # This is your preferred, seems like
```

```python
"Version {0} of package '{1}' is already "
```

```python
'Version {0} of package "{1}" is already '
```

:wink:

Reply:
Well, then sounds like  Option 1 is preferred for you indeed.